WordPress/Classes/Models/Post.m

```diff
@@ -21,6 +22,79 @@ - (void)didTurnIntoFault
     self.specialType = nil;
 }
 
+#pragma mark - Super methods
+
+- (NSArray *)availableStatuses
+{
+    NSArray *defaultStatuses = [super availableStatuses];
+    NSSet *blogStatuses = self.blog.statuses;
+    if(blogStatuses.count) {
+        
+        NSArray *filteredStatuses = @[@"auto-draft", @"inherit", @"trash"];
```

Comment:
Yup to the constants as long as those never come through localized from services.